metadata-ingestion/docs/sources/kafka/kafka.md

```diff
@@ -241,3 +241,323 @@ config:
 ```
 
 The underlying implementation is similar to [dbt meta mapping](https://docs.datahub.com/docs/generated/ingestion/sources/dbt#dbt-meta-automated-mappings), which has more detailed examples that can be used for reference.
+
+### Data Profiling
+
+The Kafka source supports comprehensive data profiling of message content to generate field-level statistics and sample values. Profiling analyzes message samples from Kafka topics to provide insights into data quality and distribution.
+
+#### Schema-less Topic Support
```

Comment:
How is schema-less section is within the data profiling section?

As per my understanding, schemaless fallback is for: in those cases where we cannot find schema in schema registry, the schema is inferred from the events themselves... and we get those events from a sampling

So schemaless is not a use case restricted to data profiling, they can be set independently

Is my understanding right?